Fix FastToLocals call by copy pasting

albanD opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #91170
* __->__ #91168
* #91166
* #91164
* #91163



cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

albanD committed:
Fix FastToLocals call by copy pasting

[ghstack-poisoned]

ezyang reviewed (approved):
Please cite the hash you copy pasted them from for ease of comparison

albanD committed:
Update on "Fix FastToLocals call by copy pasting"




cc mlazos soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen chunyuan-w XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx desertfire

[ghstack-poisoned]

albanD commented:
Added the hash.
Also added a build failure to make sure that we double check this again when 3.12 comes out.
I was considering having a new CI job that downloads the CPython source code and checks but that felt a bit overkill. Let me know if you think this is needed though.

albanD committed:
Update on "Fix FastToLocals call by copy pasting"




cc mlazos soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen chunyuan-w XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx desertfire

[ghstack-poisoned]

albanD committed:
Update on "Fix FastToLocals call by copy pasting"




cc mlazos soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen chunyuan-w XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx desertfire

[ghstack-poisoned]

albanD commented:
@pytorchbot merge
